src/Api/Dirt/Controllers/OrganizationReportsController.cs

```diff
@@ -55,11 +55,9 @@ IUpdateOrganizationReportApplicationDataCommand updateOrganizationReportApplicat
     [HttpGet("{orgId}/latest")]
     public async Task<IActionResult> GetLatestOrganizationReportAsync(Guid orgId)
     {
-        var guardResult = await GuardOrganizationAccessAsync(orgId);
-
-        if (guardResult)
+        if (!await _currentContext.AccessReports(orgId))
```

Comment:
❓ Do all these APIs need this? Can't this become an attribute on the controller?

Reply:
I'm just following a similar pattern that other sections of the API follow. I'm happy to write a new user story to make this change. Considering our deadline for this feature and the big change to the tables for this feature, we need to ship something.